coretex/cli/modules/project_utils.py

```diff
@@ -32,7 +32,7 @@ def promptProjectCreate(message: str, name: str) -> Optional[Project]:
     try:
         project = Project.createProject(name, selectedProjectType)
         ui.successEcho(f"Project \"{name}\" created successfully.")
-
+        ui.outputUrl(project.entityUrl())
```

Comment:
This should display: `A new Project has been created. You can open it by clicking on this URL ...`. A similar message should be displayed for all entities.